GitUI/Script/ScriptOptionsParser.cs

```diff
@@ -191,6 +191,7 @@ private static string AskToSpecify(IEnumerable<string> options, RevisionGridCont
                 if (revisionGrid == null)
                 {
                     var currentRevisionGuid = module.GetCurrentCheckout();
+                    currentRevision = new GitRevision(currentRevisionGuid);
```

Comment:
Why is this needed?

Reply:
> bugfix: The script options {cHash}, {cMessage}, {cAuthor}, {cCommitter}, {cAuthorDate}, {cCommitDate} did not work due to currentRevision being null when the script was not run from RevisionGridControl.

Before these script options have been added, all the existing options options starting with "c" depended on the current refs not on the commit itself.